charts/rancher-gatekeeper/102.0.0+up3.10.0/templates/namespace-post-upgrade.yaml

```diff
@@ -95,6 +95,12 @@ rules:
       {{- range .Values.postUpgrade.labelNamespace.extraNamespaces }}
       - {{ . }}
       {{- end }}
+  - apiGroups:
+      - management.cattle.io
+    resources:
+      - projects
+    verbs:
+      - updatepsa
```

Comment:
tied to broader issue with this PR referenced in https://github.com/rancher/rancher/issues/40108#issuecomment-1419569066

Reply:
addressed in https://github.com/rancher/rancher/issues/40108#issuecomment-1419597434. No more concerns with the overall approach of this PR.